cranelift/codegen/src/isa/x64/inst.isle

```diff
@@ -1829,12 +1829,18 @@
 (extern extractor is_imm8_xmm is_imm8_xmm)
 (decl is_simm8 (i8) GprMemImm)
 (extern extractor is_simm8 is_simm8)
+(decl is_rust_simm8 (i8) i32)
+(extern extractor is_rust_simm8 is_rust_simm8)
```

Comment:
We have a number of conversions along these lines in `prelude.isle` of varying names (e.g. `u8_try_from_u64`)

Could the conversions added here purely between integer types get moved over to `prelude.isle` and named similarly?

Reply:
Moved.